cmd/client-s3.go

```diff
@@ -20,7 +20,7 @@ import (
 	"bytes"
 	"context"
 	"crypto/tls"
-	"encoding/json"
+	json "github.com/minio/mc/pkg/colorjson"
```

Comment:
Not a big fan of these blanket replacements.

1) Output is not JSON and should only be used for printing.
2) Unmarshal has horrible error reporting. Even worse than stdlib.

Could you instead use this as `cjson "github.com/minio/mc/pkg/colorjson"` and only replace where it makes sense?